File: app/src/main/java/com/itachi1706/cheesecakeutilities/Modules/ListApplications/Objects/LabelledColumn.java
package com.itachi1706.cheesecakeutilities.Modules.ListApplications.Objects;

/**
 * Created by Kenneth on 1/10/2016.
 * for com.itachi1706.cheesecakeutilities.Modules.ListApplications.Objects in CheesecakeUtilities
 */

public class LabelledColumn {

    private String label, field;

    public LabelledColumn(String label, String field) {
        this.label = label;
        this.field = field;
    }

    public LabelledColumn(String label, int field) {
        this.label = label;
        this.field = field + "";
    }

    public String getLabel() {
        return label;
    }

    public String getField() {
        return field;
    }
}

File: app/src/main/java/com/itachi1706/cheesecakeutilities/modules/ListApplications/Objects/LabelledColumn.kt
package com.itachi1706.cheesecakeutilities.modules.ListApplications.Objects

/**
 * Created by Kenneth on 1/10/2016.
 * for com.itachi1706.cheesecakeutilities.Modules.ListApplications.Objects in CheesecakeUtilities
 */

data class LabelledColumn(var label: String? = null, var field: String? = null) {
    constructor(label: String, field: Int) : this(label, field.toString() + "")
}
